app/components/sections/route-plan-blueprints/route-plan-blueprint-table/component.js

```diff
@@ -14,7 +14,7 @@ export default Ember.Component.extend({
     return reg.test(routePlanBlueprint.get("name"));
   },
 
-  @computed("routePlanBlueprints.[]", "filterTerm")
+  @computed("routePlanBlueprints.@each.{name}", "filterTerm")
```

Comment:
Thanks again Aram!
I like your reviews, it helps me improve my coding skill a lot.
Review again, please